spec/features/register_names_spec.rb

```diff
@@ -0,0 +1,50 @@
+require './gamechoice'
+
+SEED = 7654
```

Comment:
Great use of SEED and srand to stub the randomness in your tests! It's really clear and concise!